dotnet/src/SemanticKernel.MetaPackage/SemanticKernel.MetaPackage.csproj

```diff
@@ -16,8 +16,10 @@ Empowers app owners to integrate cutting-edge LLM technology quickly and easily
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\SemanticKernel\SemanticKernel.csproj" />
-    <ProjectReference Include="..\Connectors\Connectors.AI.OpenAI\Connectors.AI.OpenAI.csproj" PrivateAssets="none"/>
+    <ProjectReference Include="..\SemanticKernel\SemanticKernel.csproj" PrivateAssets="none" />
+    <ProjectReference Include="..\Connectors\Connectors.AI.OpenAI\Connectors.AI.OpenAI.csproj" PrivateAssets="none" />
+    <ProjectReference Include="..\Extensions\Planning.ActionPlanner\Planning.ActionPlanner.csproj" PrivateAssets="none" />
```

Comment:
Nit: Maybe just `Planning\ActionPlanner\Planning.ActionPlanner.csproj` to follow the pattern with Skills.

Reply:
I also know we've been saying `Planning` the whole time, but maybe `Planners` makes more sense if that's all we think we will have in that namespace are various planners.

Reply:
I don't want to have an explosion of folders for everything we extend, in fact if we don't ship planners as skills, I would move skills under the extensions folder. For now I find a generic folder better scales to allow other extensions